docs/samples/deployments/data-factory/presidio-data-factory-template-gallery-databricks.md

```diff
@@ -0,0 +1,63 @@
+# Anonymize PII entities using Azure Data Factory template
+
+This sample uses the built in data anonymization template of Azure Data Factory which is a part of the Template Gallery to copy a csv dataset from one location to another while anonymizing a text column in the dataset from PII data. It leverages the code for using [Presidio on Azure Databricks](../spark/index.md) to call Presidio as a Databricks notebook job in the Azure Data Factory (ADF) pipeline to transform the input dataset before mergine the results to an Azure Blob Storage.
+
+**Note that** this solution is capabale of transforming large datasets. For smaller, text based input you may want to work with the Data Anonymization with Presidio (HTTP) template which offers an easier deployment for Presidio.
```

Comment:
can we add a link to the template or documentation here?

Reply:
done